tdm-be/src/app.ts

```diff
@@ -25,6 +25,9 @@ const limiter = rateLimit({
     limit: 2000, // Limit each IP to 1000 requests per `window` (here, per 10 minutes)
     standardHeaders: true, // Return rate limit info in the `RateLimit-*` headers
     legacyHeaders: false, // Disable the `X-RateLimit-*` headers
+    validate: {
+        trustProxy: false, // Enable been being a proxy server
```

Comment:
<!-- Nitpicks are trivial preference-based requests. These should be non-blocking by nature.  -->
**nitpick (non-blocking):** ‏I don't really understand the comment

Do you mean "Enable being behind a proxy server"?

Reply:
Yes...